aws_lambda_powertools/utilities/streaming/transformations/csv.py

```diff
@@ -0,0 +1,16 @@
+import csv
+import io
+from csv import DictReader
+from typing import IO
+
+from aws_lambda_powertools.utilities.streaming.transformations.base import BaseTransform
+
+
+class CsvTransform(BaseTransform):
```

Comment:
missing docstring to help aid customers understanding its usage and whether additional args can be used.

UX test: Customer imports it, tries to use it, what's the minimum they should know to be productive? they might not have access to the User docs.